armi/bookkeeping/db/tests/test_comparedb3.py

```diff
@@ -174,7 +174,8 @@ def test_compareDatabaseSim(self):
 
         # end-to-end validation that comparing a photocopy database works
         diffs = compareDatabases(dbs[0]._fullPath, dbs[1]._fullPath)
```

Comment:
heads up I just merged something into main affecting this line...but I don't think it should cause a merge conflict

Reply:
nevermind I gave you a merge conflict. sorry!

Reply:
thanks for the heads up!